### PR TITLE
(MAINT) Update the CHANGELOG.md for the 1.5.1 release

### PR DESCRIPTION
**Pull request by camlow325**
This commit updates the CHANGELOG.md file for the 1.5.1 release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
+## 1.5.1
```
Comment (cprice404) [inline CHANGELOG.md]: I think I like this better as a 'z' than as a 'y', but, worth raising the question.

Comment (camlow325) [inline CHANGELOG.md]: Yeah, I waffled a bit on it too.  I guess arguably it's a "new feature" and so 'y' may make more sense.  But I tend to think of it as just exposing a new way to configure an already existing feature so not sure the 'y' is totally warranted.

In the end, though, I really don't have a strong opinion about this either.

Comment (cprice404) [inline CHANGELOG.md]: Yeah.  It seems like kind of a silly thing to do a Y for so I'm good with this.  Maybe we can get one more person to weigh in, then :ship:

Comment (jpinsonault) [inline CHANGELOG.md]: I'm good with a z. Strictly speaking it's an additive backwards compatible thing, but eh

**Pull request merged**